JSDemos/Demos/Charts/AxisLabelsTemplates/description.md

```diff
@@ -0,0 +1,5 @@
+Axes [labels](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/label/) are visual elements that display values of the [major axis ticks](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/tick/).
+
+To configure labels, specify their settings in the **label** object of the [value](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/label/) or [argument](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/label/) axis. To specify options for all axes, use the **commonAxisSettings**.[label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/label/) object. Individual settings take precedence over these settings.
```

Comment:
> To configure labels, ... To specify options for all axes, ...

It is better with parallelism: "To configure labels for individual axes, ... " and "To configure labels for all axes, ...".

> the **label** object of the [value](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/label/) or [argument](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/label/) axis

We use the term "object" with label but don't do this with the axes for some reason. How about "specify label settings in the [valueAxis].[label] and [argumentAxis].[label] objects"?